tests/ResponseTest.php

```diff
@@ -111,6 +112,64 @@ public function testSetContent()
         $this->assertSame('Hello World', (string) $response->getBody());
     }
 
+    public function testSetContentWithResource()
+    {
+        $response = new Response();
+
+        $this->assertSame($response, $response->setContent(fopen('php://input', 'rb')));
+        $this->assertInstanceOf(MessageBody::class, $response->getBody());
+        $this->assertSame('', (string) $response->getBody());
+    }
+
+    public function testIsStatusCode()
+    {
+        $response = new Response();
+
+        $this->assertTrue($response->isStatusCode(200));
+        $this->assertFalse($response->isStatusCode(400));
+    }
+
+    /**
+     * @expectedException        \RuntimeException
+     * @expectedExceptionMessage Could not parse response (error 1).
+     */
+    public function testParseShouldReturnRuntimeExceptionError1()
+    {
+        Response::parse('<!DCOTYPE html><html>HTML strings</html>');
+    }
+
+    /**
+     * @expectedException        \RuntimeException
+     * @expectedExceptionMessage Could not parse response (error 2).
+     */
+    public function testParseShouldReturnRuntimeExceptionError2()
+    {
+        Response::parse('HTTP/1.0 200 OK' . "\r\n" . 'Content-Length: 20' . "\r\n");
+    }
+
+    /**
+     * @expectedException        \RuntimeException
+     * @expectedExceptionMessage Could not parse response (error 3).
+     */
+    public function testParseShouldReturnRuntimeExceptionError3()
+    {
+        Response::parse('HTTP/1.0 200 OK' . "\r\n" . 'Content-Length20' . "\r\n\r\n");
+    }
+
+    public function testParseShouldReturnResponseObject()
+    {
+        $result = Response::parse('HTTP/1.0 200 OK' . "\r\n" . 'Content-Length: 20' . "\r\n\r\n");
+
+        $this->assertInstanceOf(Response::class, $result);
```

Comment:
It would be good to test getters: protocol version, response code, headers... Not just that a Response is returned!